Portfolio.html

```diff
@@ -34,32 +36,32 @@ <h1>JENNY ALLTON-TWI<b>S</b>T</h1>
 	<div class="portfolio" id="portfolio"><h2>Portfolio</h2>
 		<div class="container">
 			<div class="box">
-				<a href="#"><img src="tech.jpg" alt="Project Image"/></a>
+				<a href="project.html"><img src="tech.jpg" alt="Project Image"/></a>
 				<p>Some info about this very cool thing I invented here</p>
 				<h4>October 2020, Mayden Academy, Bath UK</h4>
```

Comment:
These h4 tags are being displayed underneath of their box container, so overflowing onto the box below

Reply:
Thank you - I thought I had fixed that. I'll look.

Reply:
Sorted!

Reply:
Its better but still a bit of overlap in Safari & Chrome
<img width="1420" alt="Screenshot 2020-09-08 at 14 09 01" src="https://user-images.githubusercontent.com/65677163/92480535-eed11380-f1dc-11ea-91eb-9d0de2278d2a.png">

Reply:
All good